src/components/Employee/Deductions/IncludeDeductionsForm/IncludeDeductionsFormEmptyState.tsx

```diff
@@ -0,0 +1,103 @@
+import { useTranslation } from 'react-i18next'
+import { z } from 'zod'
+import { FormProvider, useForm, type SubmitHandler } from 'react-hook-form'
+import { zodResolver } from '@hookform/resolvers/zod'
+import styles from './IncludeDeductionsFormEmptyState.module.scss'
+import {
+  BaseComponent,
+  type BaseComponentInterface,
+  type CommonComponentInterface,
+  useBase,
+} from '@/components/Base'
+import { Form } from '@/components/Common/Form'
+import { Grid } from '@/components/Common/Grid/Grid'
+import { ActionsLayout, Flex } from '@/components/Common'
+import { useI18n } from '@/i18n'
+import { componentEvents } from '@/shared/constants'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import { useComponentDictionary } from '@/i18n/I18n'
+import CoinsHandsIcon from '@/assets/icons/coins-hand.svg?react'
+import PlusCircleIcon from '@/assets/icons/plus-circle.svg?react'
+
+const IncludeDeductionsSchema = z.object({ includeDeductions: z.enum(['Yes', 'No']) })
+export type IncludeDeductionsPayload = z.output<typeof IncludeDeductionsSchema>
+
+interface IncludeDeductionsFormProps extends CommonComponentInterface<'Employee.Deductions'> {
+  employeeId: string
+}
+
+export function IncludeDeductionsFormEmptyState(
```

Comment:
I'm a little confused on the naming with the implementation here? If this is just an empty state component it can probably just include the empty state UI pieces without needing to implement a schema, form, or the base component.

If this is eventually going to grow in functionality to contain the form, we should probably update the file naming.

Reply:
renamed component to EmptyState to better match functionality and removed form methods as not required